.gitlab/source_test/windows.yml

```diff
@@ -24,6 +24,7 @@
       -e GITLAB_CI=true
       -e CI_JOB_URL="${CI_JOB_URL}"
       -e CI_JOB_NAME="${CI_JOB_NAME}"
+      -e CI_JOB_ID="${CI_JOB_ID}"
```

Comment:
I'm adding it to be correctly forwarded to Codecov through its uploader. Since the unit tests is the only job where the codecov upload is running, it's not needed in other Windows jobs !